packages/ListBoxBrowser/src/ListBoxBrowser.js

```diff
@@ -95,6 +95,10 @@ const propTypes = {
     initial value.
   */
   defaultSelectedView: PropTypes.func,
+  /** 
+    In the case you want to use the ListBoxBrowser with one column you can hide the root column
+   */
+  isVisibleRoot: PropTypes.bool,
```

Comment:
`isRootVisible` sounds better in English, I think. But maybe `hasLeftColumn` would be more obvious?

Reply:
I know but didn't we agree to start always with isVisible => then what?

Reply:
taking hasLeftColumn then